JS/test/airport.spec.js

```diff
@@ -20,85 +20,51 @@ describe('My Test', () => {
         new PassengerPlane('Embraer 190', 870, 8100, 30800, 64),
         new PassengerPlane('Sukhoi Superjet 100', 870, 11500, 50500, 140),
         new PassengerPlane('Bombardier CS300', 920, 11000, 60700, 196),
-        new MilitaryPlane('B-1B Lancer', 1050, 21000, 80000, MilitaryType.BOMBER),
-        new MilitaryPlane('B-2 Spirit', 1030, 22000, 70000, MilitaryType.BOMBER),
-        new MilitaryPlane('B-52 Stratofortress', 1000, 20000, 80000, MilitaryType.BOMBER),
-        new MilitaryPlane('F-15', 1500, 12000, 10000, MilitaryType.FIGHTER),
-        new MilitaryPlane('F-22', 1550, 13000, 11000, MilitaryType.FIGHTER),
-        new MilitaryPlane('C-130 Hercules', 650, 5000, 110000, MilitaryType.TRANSPORT),
-        new experimentalPlane("Bell X-14", 277, 482, 500, ExperimentalTypes.HIGH_ALTITUDE, ClassificationLevel.SECRET),
-        new experimentalPlane("Ryan X-13 Vertijet", 560, 307, 500, ExperimentalTypes.VTOL, ClassificationLevel.TOP_SECRET)
+        new MilitaryPlane('B-1B Lancer', 1050, 21000, 80000, militaryType.bomber),
+        new MilitaryPlane('B-2 Spirit', 1030, 22000, 70000, militaryType.bomber),
+        new MilitaryPlane('B-52 Stratofortress', 1000, 20000, 80000, militaryType.fighter),
+        new MilitaryPlane('F-15', 1500, 12000, 10000, militaryType.fighter),
+        new MilitaryPlane('F-22', 1550, 13000, 11000, militaryType.fighter),
+        new MilitaryPlane('C-130 Hercules', 650, 5000, 110000, militaryType.transport),
+        new experimentalPlane("Bell X-14", 277, 482, 500, experimentalTypes.altitude_limit, classificationLevel.secret),
+        new experimentalPlane("Ryan X-13 Vertijet", 560, 307, 500, experimentalTypes.take_off_type, classificationLevel.top_secret)
     ];
-    let planeWithMaxPassengerCapacity = new PassengerPlane('Boeing-747', 980, 16100, 70500, 242);
 
-    it('should have military Planes with transport type', () => {
-        let airport = new Airport(planes);
-        let transportMilitaryPlanes = airport.getTransportMilitaryPlanes();
-        let flag = false;
-        for (let militaryPlane of transportMilitaryPlanes) {
-            if (militaryPlane.getMilitaryType() === MilitaryType.TYPE_TRANSPORT) {
-                flag = true;
-                break;
-            }
+    it('Should check that at least one military transport plane is present', () => {
+        if (new Airport(planes).getTransportMilitaryPlanes().length) {
+            return true;
```

Comment:
`return` keyword will make your it to be green. It means that your test is "falsy positive". Please correct.